com.unity.multiplayer.mlapi/Prototyping/NetworkAnimator.cs

```diff
@@ -238,11 +239,17 @@ public enum Authority
             Server = 0,
             Owner
         }
+        
+        /// <summary>
+        /// This constant is used to force the resync if the delta between current
+        /// and last synced normalized state time goes above it
+        /// </summary>
+        private const float NormalizedTimeResyncThreshold = 0.15f;
 
         /// <summary>
         /// Specifies who can update this animator
         /// </summary>
-        [Tooltip("Defines who can update this transform.")]
+        [Tooltip("Defines who can update this animator.")]
         public Authority AnimatorAuthority = Authority.Owner;
```

Comment:
hopefully, we'll figure out SDK-wide permissions, ownership and authority concepts one day :)